main.tf

```diff
@@ -71,6 +71,38 @@ resource "aws_vpn_connection" "default" {
   tunnel1_ike_versions = var.tunnel1_ike_versions
   tunnel2_ike_versions = var.tunnel2_ike_versions
 
+  dynamic "tunnel1_log_options" {
+    for_each = [var.tunnel1_log_options]
+
+    content {
+      dynamic "cloudwatch_log_options" {
+        for_each = tunnel1_log_options.value
+
+        content {
+          log_enabled       = cloudwatch_log_options.value.log_enabled
```

Comment:
All of these `cloudwatch_log_options` options are optional, so you should use `lookup(cloudwatch_log_options.value, "log_enabled", null)`. Same in other optional arguments.

Reply:
Done 👍